core/src/impl/Kokkos_Error.hpp

```diff
@@ -20,82 +20,14 @@
 #include <string>
 #include <iosfwd>
 #include <Kokkos_Macros.hpp>
```

Comment:
```suggestion
```

Reply:
If I don't include that, where does `KOKKOS_ENABLE_DEPRECATED_CODE_4` come from?

Reply:
I overlooked this.  This is fine to include and to rely on transitive include.